.github/ISSUE_TEMPLATE/1_feature_template.md

```diff
@@ -10,15 +10,29 @@ What is the purpose, what´s the expected result.
 Please describe.
 -->
 
-## Impact
+## Impact/Dependencies
 <!--
 If possible, please provide insight in what and how many components and teams might be affected or give a hint, if the feature implementation might result in breaking changes.
 If there are dependencies to other features please name it.
 -->
 
-## Additional information 
-
-- [ ] I'm willing to contribute to this feature
+## Acceptance Criteria
+<!--
+Add all Acceptence Criteria. These criteria are important for the definition of done
+-->
+- [ ] Criteria 1
+- [ ] Criteria 2
+- [ ] Criteria 3
 
+## Linked Subtasks/Product specific tasks
+<!--
+PLease link all related subtasks which are need to fullfill the requirements for this featue. If you add a URL from a different issue (of a different repository) this will be interpreted by GitHub automaticly
+-->
+- [ ] Issue 1 to product repository
+- [ ] Issue 2 to product repository
+- [ ] Issue 3 to product repository
 
+## Additional information 
 
+- [ ] I'm willing to contribute to this feature
```

Comment:
Question: How to deal with this checkbox, in case that more than one person is willing to contribute - which is hopefully the standard case in the future :-)

Reply:
I guess depends on the working model, but if the person who wrote the issue is willing to contribute and want to add more people to this issue ... then the writer can easily change the description within the template "we are willing to contribute"